### PR TITLE
TA-265: Contracted Work and Administration "operational" system context diagram, phase 1

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

Provides a new diagram with for the systems and teams who are involved in processes touching the **Contracted Work and Administration** (CWA) application. We will treat this application as a system.

This is **phase one**, where we document the system-to-system interactions.
**Phase two** will document the team-to-system interactions (how people use the system).

:memo: Interactions through the Hub are marked with a “via-hub” arrow style. I found it more useful to treat the Hub as a protocol than to treat it as a system (similar to RabbitMQ).

## Why make these changes?

There is currently no diagram in this repository which presents the business processes and directly integrated systems for this application.

### CODE REVIEW

**Comment by ddippolito** [comment]
Did you purposely not mention the Hub among the interactions?

File: diagrams/get-access/cwa-system-context.yaml [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  position: '225,2050'
+- type: Person
+  name: Operational Assurance Team
+  position: '225,2750'
+- type: Person
+  name: Provider Records Team
+  position: '1225,2750'
+- type: Person
+  name: Reconciliation Team
+  description: In reality, only one person works on this from the team, Nigel.
```
Comment (ddippolito) [inline diagrams/get-access/cwa-system-context.yaml]: TBC

**Comment by ddippolito** [comment]
If Phase one is about system to system interaction is worth removing the teams from this diagram?
Or did you leave them there as a placeholder?

**Comment by sldblog** [comment]
> If Phase one is about system to system interaction is worth removing the teams from this diagram?
> Or did you leave them there as a placeholder?

I think at this point it's useful to have them as a placeholder, as it stresses that the diagram is incomplete. What do you think?

**Comment by sldblog** [comment]
📝 Rebased on top of `master` to get #17's changes.

**Comment by sldblog** [comment]
🎩 🙇 For all your help reviewing this PR!

**Pull request merged**